print_pageids_from_wikipedia.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+_upper_dir = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '..'))
+if _upper_dir not in sys.path:
+    sys.path.append(_upper_dir)
+
+import config
+import database
+
+def print_pageids_from_wikipedia():
+    cfg = config.get_localized_config()
+    db = database.init_wp_replica_db(cfg.lang_code)
+    cursor = db.cursor()
+
+    cursor.execute(
+        'SELECT page_id FROM page where page_namespace=0 LIMIT %s' % int(cfg.num_pages * cfg.X * 0.01))
```

Comment:
Rather than hard-coding the number of pages, which will inevitable go out of date, how about something like:

```sql
SELECT page_id FROM page WHERE page_namespace = 0 AND RAND() < cfg.articles_sampling_fraction
```

http://bigdatums.net/2017/01/07/select-n-percent-records-mysql/